### PR TITLE
DWR-1109 improvements to original bug

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
I'm in favor of moving "Remove District" to some other file (like en.json), I just don't know how to do that

**EDIT**
moved "Remove District" to en.json

on hover
screen shot 2017-10-27 at 12 53 51 pm

### CODE REVIEW

File: webapp/src/main/webapp/src/app/organization-export/organization/organization-tree.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <div *ngFor="let districtTree of model.children" class="district-tree">
 
   <a *ngIf="districtTree.value.name && !disabled" class="link-button" (click)="onClick(districtTree.value)" href="javascript:void(0)">
-    <h3>{{districtTree.value.name}} <span class="icon"><i class="fa fa-close"></i></span></h3>
+    <h3>{{districtTree.value.name}}<a class="tag tag-xs"><span class="pr-xs"></span> Remove District <span class="icon"><i class="fa fa-close"></i></span></a></h3>
```
Comment (jtreuting) [inline webapp/src/main/webapp/src/app/organization-export/organization/organization-tree.component.html]: Looks like you have an `<a>` inside an `<a>` which doesn't feel right.

To change the text for internationalization you would add Remove District to the en.json file and then replace it with something like `{{'labels.something.blah.remove-district' | translate}}`

File: webapp/src/main/webapp/src/app/organization-export/organization/organization-tree.component.ts [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 import { Tree } from "./tree";
+import { TranslateService } from "@ngx-translate/core";
 
 @Component({
   selector: 'organization-tree',
   templateUrl: './organization-tree.component.html'
 })
 export class OrganizationTreeComponent {
 
+  constructor (private translate: TranslateService) {}
```
Comment (wtritch) [inline webapp/src/main/webapp/src/app/organization-export/organization/organization-tree.component.ts]: I don't think you need to inject this in here unless you're planning on using it programmatically.

File: webapp/src/main/webapp/src/styles.less [hunk 3]
```diff
@@ -116,6 +116,12 @@ footer {
   z-index: 1030;
 }
 
+.district-tree > .tag {
+  margin-left: @grid-gutter-width / 2;
+  position: absolute;
```
Comment (bgsavage) [inline webapp/src/main/webapp/src/styles.less]: Hm, is position: absolute needed?

**Comment by esotrope** [comment]
Hey @dsnyder-fairway, do you mind adding a screenshot of the new view to the description. It helps me a lot when reviewing ui changes - thanks 👍

File: webapp/src/main/webapp/src/app/organization-export/organization/organization-tree.component.ts (hunk 2 above)
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/app/organization-export/organization/organization-tree.component.ts]: removed

File: webapp/src/main/webapp/src/styles.less (hunk 3 above)
Comment (dsnyder-fairway) [reply webapp/src/main/webapp/src/styles.less]: removed

**Pull request merged**